### ISSUE TITLE
xAxisPaddingUnit default value is not 'days'

### ISSUE THREAD

**Issue by a0th** [dc-js/dc.js#1326]
Hey guys, not really sure where to report this, but here it goes anyway.
The documentations for both the 2.1.6 and 2.1.8 say the default xAxisPaddingUnit value is 'days'. 
I dont think that is true: setting it manually to 'days' raises the error 

```
Uncaught TypeError: d3.time[t].offset is not a function
    at Object.dc.utils.subtract (utils.js:186)
    at Object._chart.xAxisMin (stack-mixin.js:196)
    at prepareXAxis (coordinate-grid-mixin.js:459)
    at drawChart (coordinate-grid-mixin.js:1139)
    at Object._chart._doRender (coordinate-grid-mixin.js:1118)
    at Object._chart.render [as _render] (base-mixin.js:660)
    at Object.render (bar-chart.js:52)
```

Thats also true using 'months'. Using 'day' and 'month', however, work as intended.

**Comment by gordonwoodhull** [comment]
You're right, it's a typo in the documentation; the default is `'day'`. Will fix.

**Comment by gordonwoodhull** [comment]
Fixed in 3.0 alpha 11
